pipeline/pipelines/code_generation_pipeline.py

```diff
@@ -236,8 +236,9 @@ def __init__(self, **kwargs):
 
     def do_build_flow(self, **kwargs):
         flow = linear_flow.Flow('grpc-codegen')
-        flow.add(protoc_tasks.ProtoCodeGenTask('ProtoGen', inject=kwargs),
-                 protoc_tasks.GrpcCodeGenTask('GrpcCodegen', inject=kwargs))
+        kwargs.update({'packman_flags': ['--experimental_alt_java']})
+        flow.add(protoc_tasks.GrpcPackmanTask('Packman', inject=kwargs),
+                 package_tasks.GrpcPackageDirTask('PackageDir', inject=kwargs))
```

Comment:
I think the original intention for GrpcPackageDirTask is to prepare the package_dir so that one can run package commands. For example, running `setup.py bdist_wheel` inside package_dir to generate whl for Python. @geigerj Can you confirm, Jacob?

Since we don't generate any extra package file for Java, then we don't need this GrpcPackageDirTask.